core/CHANGELOG.T3T1.md

```
@@ -1361,6 +1380,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 [#6165]: https://github.com/trezor/trezor-firmware/pull/6165
 [#6186]: https://github.com/trezor/trezor-firmware/pull/6186
 [#6202]: https://github.com/trezor/trezor-firmware/pull/6202
+[#6211]: https://github.com/trezor/trezor-firmware/pull/6211
```

markdownlint-cli2 (0.23.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "`#6211`"

(MD053, link-image-reference-definitions)